cmd/config/internal/commands/cmdset.go

```diff
@@ -30,6 +30,9 @@ func NewSetRunner(parent string) *SetRunner {
 	}
 	fixDocs(parent, c)
 	r.Command = c
+	c.Flags().StringVar(&r.Set.Name, "name", "", "name of the setter")
```

Comment:
Name should be an argument and not flag. We don't have a need currently to change it to flag. Any reason why it should be a flag? This name is created by user and it wouldn't start with -

Reply:
reverted, was thinking to convert all three arguments to flags